File: Eldar Dautović/ridewithme/ridewithme.Model/Models/KorisniciDostignuca.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ridewithme.Model.Models
{
    public class KorisniciDostignuca
    {
        public int Id { get; set; }

        public virtual Dostignuca Dostignuce { get; set; }

        public DateTime DatumKreiranja { get; set; }
    }
}

File: Eldar Dautović/ridewithme/UI/ridewithme_mobile/lib/models/korisnici_dostignuca.dart
// -- korisnici_dostignuca.dart --
import 'package:json_annotation/json_annotation.dart';
import 'package:ridewithme_mobile/models/dostignuca.dart';

part 'korisnici_dostignuca.g.dart';

@JsonSerializable()
class KorisniciDostignuca {
  int? id;
  Dostignuca? dostignuce;
  DateTime? datumKreiranja;

  KorisniciDostignuca(
    this.id,
    this.dostignuce,
    this.datumKreiranja,
  );

  factory KorisniciDostignuca.fromJson(Map<String, dynamic> json) =>
      _$KorisniciDostignucaFromJson(json);

  Map<String, dynamic> toJson() => _$KorisniciDostignucaToJson(this);
}
